### ISSUE TITLE
pre-commit hadolint-docker pulls old docker image for linux/arm64 (macOS M1)

### ISSUE THREAD

**Issue by 1uk4sh** [hadolint/hadolint#840]
- [x] This is a bug report
- ❌  This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
pre-commit hadolint-docker pulls image `.../hadolint:v2.10.0` if specified

### Actual behavior
pre-commit hadolint-docker pulls image `ghcr.io/hadolint/hadolint:v2.9.3`

### Steps to reproduce the behavior

include `hadolint-docker` in your .pre-commit-hook.yaml
```yaml
---
repos:
  - repo: https://github.com/hadolint/hadolint
    rev: v2.10.0
    hooks:
      - id: hadolint-docker
        args: [--config, hadolint.yaml]
 ```

run example command to get pre-commit started
e.g. `pre-commit run --all-files`

```bash
[...]
Lint Dockerfiles.........................................................Failed
- hook id: hadolint-docker
- exit code: 137

Unable to find image 'ghcr.io/hadolint/hadolint:v2.9.3' locally
v2.9.3: Pulling from hadolint/hadolint
4022150d876a: Pull complete
Digest: sha256:c90fe0d22b6aa7cb8114cd4022cf7a596f61724e2c66b4e0b03435326f29dede
Status: Downloaded newer image for ghcr.io/hadolint/hadolint:v2.9.3
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
# docker run --rm hadolint/hadolint hadolint --version
Haskell Dockerfile Linter 2.10.0
# docker run --rm ghcr.io/hadolint/hadolint hadolint --version
Haskell Dockerfile Linter 2.10.0
```

**Additional environment details (OS, stack version, etc.)**
```
System Version:	macOS 12.3 (21E230)
Kernel Version:	Darwin 21.4.0
Model Name:	MacBook Pro
Chip:           Apple M1 Max
```

**Comment by 1uk4sh** [comment]
already fixed by #814 
any chance to bump to `v2.10.1` soon?
